jormungandr-watchdog/src/service/mod.rs

```diff
@@ -145,8 +145,8 @@ impl<T: Service> ServiceState<T> {
     /// this from there one can "borrow" the settings or clone the reader
     ///
     /// [`SettingsReader`]: ./struct.SettingsReader.html
-    pub fn settings(&self) -> &SettingsReader<T::Settings> {
-        &self.settings
+    pub fn settings(&mut self) -> &mut SettingsReader<T::Settings> {
```

Comment:
I had to change this reference. Otherwise, it is impossible to use the `updated` method over the settings.

Reply:
`SettingsReader` is clonable: https://github.com/input-output-hk/jormungandr/blob/40c120957a00e042060df90071303e024c1da731/jormungandr-watchdog/src/service/settings.rs#L90-L94